kernelci/api/models.py

```diff
@@ -47,6 +47,12 @@ class ResultValues(str, enum.Enum):
     INCOMPLETE = 'incomplete'
 
 
+class FailValues(str, enum.Enum):
```

Comment:
I assume these will contain different error codes received from the lava job run.
Using `ErrorCodes` and `Node.error_code` seem more suitable to me.

Reply:
done